drivers/sensor/bmi270/Kconfig

```diff
@@ -5,8 +5,6 @@
 
 config BMI270
 	bool "BMI270 Inertial measurement unit"
-	default y
-	depends on DT_HAS_BOSCH_BMI270_ENABLED
-	select I2C
+	depends on I2C
```

Comment:
The point of the PR is adding SPI support for the sensor, but you want to add `depends on I2C`?